app/src/main/java/org/systers/mentorship/viewmodels/TaskViewModel.kt

```diff
@@ -71,26 +76,64 @@ class TasksViewModel: ViewModel() {
     }
 
     /**
-     * This function helps in adds a new task to the task list
-     * @param taskName title of the new task
+     * This function adds a new task to the task list
+     * If the operation is successful, [getTasks] is automatically triggered
+     * @param description description/title of the new task
      */
-    fun addTask(taskName: String) {
-        //TODO: Update the backend
+    @SuppressLint("CheckResult")
+    fun addTask(description: String) {
+        val taskRequest = TaskRequest(description)
+
+        // TODO
```

Comment:
Please remove this implementation inside **addTask** method. Replace this with TODO comment that was there before - **//TODO: Update the backend**